### PR TITLE
gui/app context menu: rename looks like more reasonable.

### PR DESCRIPTION
**Pull request by Croden1999**
# About
- the ‘DLC’ rename to ‘AddCont’ looks like more reasonable, and the 'AddCont' option is based on psvita devkit / testkit system.

### CODE REVIEW

File: vita3k/gui/src/content_manager.cpp [hunk 1]
```diff
@@ -185,43 +185,43 @@ static bool get_size_selected_contents(GuiState &gui, HostState &host) {
     return contents_size;
 }
 
-struct Dlc {
+struct addcont {
```
Comment (Zangetsu38) [inline vita3k/gui/src/content_manager.cpp:188]: fail lol, struct is all time first letter in shift

**Comment by Zangetsu38** [comment]
you need learn git, for edit last commit, rebase, force push

File: vita3k/gui/src/content_manager.cpp (hunk 1 above)
Comment (Croden1999) [reply vita3k/gui/src/content_manager.cpp:188]: i got it.😭

Comment (Zangetsu38) [reply vita3k/gui/src/content_manager.cpp:188]: so here is AddCont

**Comment by Zangetsu38** [comment]
PLEASE, squash, no create new commit any time lol, using git and tortoise git, is very easy 👌

**Comment by Croden1999** [comment]
> PLEASE, squash, no create new commit any time lol, using git and tortoise git, is very easy 👌

thx a lot again, Mr. Zangetsu.

**Comment by Zangetsu38** [comment]
Rebase on master please

**Comment by Croden1999** [comment]
> Rebase on master please



> Rebase on master please

i got it, sir.

**Comment by Zangetsu38** [comment]
> > Rebase on master please
> 
> i got it, sir.

Fail lol using git rebase

**Comment by Zangetsu38** [comment]
Keep fail, i sais using rebase, here you have added New master commit in your pr, is not rebase

**Comment by Zangetsu38** [comment]
NO close again.... no open new pr any time, LEARN git

**Comment by Croden1999** [comment]
> and is not possible reopen pr after your force push, just jkeep pr open and learn how cor(reectly using git :/

my mistake, Mr. Zangetsu, i will to learn the knowledge of Git, and how use this.

**Review by Zangetsu38 — APPROVED** [review]

**Comment by hobyst** [comment]
The `addcont` / _AddCont_ nomenclature is only used towards developers. For actual end users of the system the term _DLC_ / _downloadable content_ has always been used by Sony and other developers.
dlc

**Comment by Zangetsu38** [comment]
> The `addcont` / _AddCont_ nomenclature is only used towards developers. For actual end users of the system the term _DLC_ / _downloadable content_ has always been used by Sony and other developers. dlc

ah yes, that is true 🤔

**Comment by Croden1999** [comment]
> > The `addcont` / _AddCont_ nomenclature is only used towards developers. For actual end users of the system the term _DLC_ / _downloadable content_ has always been used by Sony and other developers. dlc
> 
> ah yes, that is true thinking

i bought a Testkit motherboard on last year, Mr. Zangetsu. so i want to rename the strings to reasonable.

**Comment by Zangetsu38** [comment]
> > > The `addcont` / _AddCont_ nomenclature is only used towards developers. For actual end users of the system the term _DLC_ / _downloadable content_ has always been used by Sony and other developers. dlc
> > 
> > 
> > ah yes, that is true thinking
> 
> i bought a Testkit motherboard on last year, Mr. Zangetsu. so i want to rename the strings to reasonable.
> 
> 
> 
> 

yes, but probleme, test/dev kit is diference release;, for User is writed Dlc

**Comment by Croden1999** [comment]
> > > > The `addcont` / _AddCont_ nomenclature is only used towards developers. For actual end users of the system the term _DLC_ / _downloadable content_ has always been used by Sony and other developers. dlc
> > > 
> > > 
> > > ah yes, that is true thinking
> > 
> > 
> > i bought a Testkit motherboard on last year, Mr. Zangetsu. so i want to rename the strings to reasonable.
> > 
> > 
> 
> yes, but probleme, test/dev kit is diference release;, for User is writed Dlc

the retail psvita(no henkaku) is could not delete only all dlc(add-on), should based on testkit/debkit option, there have complete language in add-on data. such 'add-on data' -> '追加資料'.

**Pull request merged**